### ISSUE TITLE
Error trying to start a logs filter

### ISSUE THREAD

**Issue by whyrusleeping** [ethereum/go-ethereum#3001]
#### System information

Geth version: 1.5.0-unstable
OS & Version: linux
Commit hash : 7c12e8ea44129a5f9caa4d5ca9168c059436bb39

I'm trying to get information about events that my contract is emitting, so I wrote some code that looks like:

```go
        fq := ethereum.FilterQuery{
                FromBlock: big.NewInt(0),
                Addresses: []common.Address{regaddr},
        }

        logs, err := ec.FilterLogs(context.Background(), fq)
        if err != nil {
                panic(err)
        }
```

And the `FilterLogs` call errors with: "json: cannot unmarshal object into Go value of type rpc.ID"

**Comment by fjl** [comment]
Fixed in #3021

**Comment by fjl** [comment]
Note that you can leave off FromBlock because it defaults to zero when nil.
